test/unit/winston/transports/console.test.js

```diff
@@ -139,6 +139,7 @@ require('abstract-winston-transport')({
   Transport: winston.transports.Console
 });
 
+// TODO(invalid-test): test is no longer valid as we don't have the vows dependency anymore
```

Comment:
Example bad test that needs addressed at some point.

Reply:
Why add a comment to already commented out code? I would just say remove the commented out code instead.

Reply:
Introduction of a TODO for myself (or anyone else) to assist in finding again so the test can be gone through to see if it has an equivalent test already, or if it needs to be rewritten with `mocha/assume`. I'm happy to remove all of it if you want.